index.html

```diff
@@ -0,0 +1,205 @@
+      <h2>Expert Panel and Q&A Sessions</h2>
+     <a href="#seemore">SEE MORE</a>
+    </div>
+  </div>
+  <div class="joinUs "><a href="">Reserve Your Spot Now</a> </div>
+  <div class="lecture"></div>
+</section>
+
+<section class="section3 Feature-speakers">
+  <h2>Founders</h2>
+  <div class="line"></div>
+  <div class="speakers">
+
+<div class="person ">
+    <img class="personImage"  src="images/peter2.jpg"alt="peter_image">
+  <div class="clm2">
+    <h2 class="name">Peter</h2>
+     <p class="p1">Peter, our accomplished CEO, leads with vision and expertise, propelling our company to new heights of success. His strategic acumen and commitment to innovation drive our team to deliver exceptional results.</p>
+     <div class="line2"></div>
+     <p class="p2">Chief Executive Officer (CEO)</p>
+  </div>
+</div>
+
+<div class="person">
+  <img class="personImage"  src="images/mayssa5.webp"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Mayssa</h2>
+   <p class="p1"> Responsible for developing and executing the company's marketing strategies. </p>
+   <div class="line2"></div>
+   <p class="p2">Chief Marketing Officer (CMO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/santiago3.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Santiago</h2>
+   <p class="p1">Manages the company's financial operations, including budgeting, financial planning, and financial reporting..</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Financial Officer (CFO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/catherine4.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Catherine</h2>
+   <p class="p1">Oversees the day-to-day operations of the company, ensuring efficient and effective processes across various departments.</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Operations Officer (COO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/stellar1.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Stellar</h2>
+   <p class="p1">Responsible for aligning technology initiatives with business objectives, managing IT infrastructure, and driving digital transformation.</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Technology Officer (CTO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/houda6.jpg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Houda</h2>
+   <p class="p1">Oversees all aspects of human resources, including recruitment, employee relations, training and development, and HR policies. </p>
+   <div class="line2"></div>
+   <p class="p2">Human Resources Director</p>
+</div>
+</div>
+
+<div id="more" class="more">MORE
+  <img src="images/arrow_down 1.png" alt="arrow-icon">
+</div>
+<div id="less" class="more">LESS
+  <img src="images/arrow_up.png" alt="arrow-icon">
+</div>
+</div>
+
+</section>
```

Comment:
Kindly implement this section dynamically using javascript as stated in the requirement :

![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/d4302a06-65e1-4c67-97ee-0fd13e2bd321)